core/client/hdfs/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -259,10 +259,11 @@ public long getDefaultBlockSize() {
   @Override
   public BlockLocation[] getFileBlockLocations(FileStatus file, long start, long len)
       throws IOException {
-    LOG.debug("getFileBlockLocations({}, {}, {})", file.getPath().getName(), start, len);
+    LOG.debug("getFileBlockLocations({}, {}, {})",
```

Comment:
shouldn't this just be the same as old code, but moved to after the `null` check?

Reply:
Neh. Because it seems like there is a debug message for when the code returns AND when the code gets called. (Note the LOG.debug further down this code)